### DISCUSSION TITLE
How is this going to be better than radio-browser.info?

### DISCUSSION THREAD

**Discussion by pirateradiohack** [jcorporation/webradiodb#405] [General]
I fully support efforts to get web radios streams open databases. I think it helps everybody and is a great endeavor. 

I think duplicating efforts can hinder global quality or on the opposite introduce some creativity. 

In this case I wonder what this database aims to bring that radio-browser.info does not support or would not benefit from some contributions.

In another issue (https://github.com/jcorporation/myMPD/issues/933) @jcorporation has already started to answer that radio-browser.info has a "big quality problem, because of this: https://github.com/segler-alex/radiobrowser-api-rust/issues/49".
And invites me to start a new discussion if I'm interested.

So, yes, I'm curious (and I wonder which one to use also!), how are the goals of webradiodb different from those of radio-browser.info. And if the editing part is a big issue for radio-browser.info, how webradiodb deals with it?

Thanks very much!

**Comment by jcorporation** [comment]
I started the WebradioDB project after integrating the radio-browser.info project into myMPD. I do not know if the myMPD uses the WebradioDB or the radio-browser.info project more.

I was personally not satisfied with the provided search results, lots off duplicate entries that point to the same radio station. Let me illustrate my experience:

Search for the german SWR 1 radio station, results:
1. SWR1 BW
2. SWR1 BW neu (2021.12)
3. SWR1 BW neu
4. SWR1 Baden Württemberg 
5. SWR1 - BW | 48 aac

Five search results for the exact same station and there is no possibility to fix this through a contribution effort, because of the disabled editing feature.

WebradioDB fixes this problem by accepting new webradios, deletions and modifications only through GitHub issues. There are workflows behind, that must be manually approved.

**Comment by pirateradiohack** [comment]
Thanks for explaining, very interesting. 

So I understand that by using GitHub issues, the contributions are publicly traceable. And on top of that you will approve them manually. 
Makes sense. I hope you receive some good contributions and have the time to triage them.

**Reply by jcorporation** [reply]
Everbody is welcome to improve the database. If there are some time in the future to many issues, volunteers can help approving changes. This is an open project.
